crates/bevy_reflect/src/type_registry.rs

```diff
@@ -234,6 +234,32 @@ impl TypeRegistry {
             .and_then(|id| self.registrations.get_mut(id))
     }
 
+    /// Returns `true` if the given [short type path] is ambiguous, that is, it matches multiple registered types.
+    ///
+    /// # Example
+    /// ```
+    /// # use bevy_reflect::TypeRegistry;
+    /// # mod foo {
+    /// #     use bevy_reflect::Reflect;
+    /// #     #[derive(Reflect)]
+    /// #     struct MyType;
+    /// # }
+    /// # mod bar {
+    /// #     use bevy_reflect::Reflect;
+    /// #     #[derive(Reflect)]
+    /// #     struct MyType;
```

Comment:
```suggestion
    /// #     pub struct MyType;
    /// # }
    /// # mod bar {
    /// #     use bevy_reflect::Reflect;
    /// #     #[derive(Reflect)]
    /// #     pub struct MyType;
```

Reply:
Thanks! TBH I should have just looked at the CI error messages before 😅